cluster/gce/gci/apiserver_manifest_test.go

```diff
@@ -51,89 +55,138 @@ readonly DOCKER_REGISTRY="k8s.gcr.io"
 readonly ENABLE_LEGACY_ABAC=false
 readonly ETC_MANIFESTS=${KUBE_HOME}/etc/kubernetes/manifests
 readonly KUBE_API_SERVER_DOCKER_TAG=v1.11.0-alpha.0.1808_3c7452dc11645d-dirty
-readonly LOG_OWNER_USER=$(id -un)
+readonly LOG_OWNER_USER=$(whoami)
 readonly LOG_OWNER_GROUP=$(id -gn)
+{{if .EncryptionProviderConfig}}
 ENCRYPTION_PROVIDER_CONFIG={{.EncryptionProviderConfig}}
+{{end}}
 ENCRYPTION_PROVIDER_CONFIG_PATH={{.EncryptionProviderConfigPath}}
-readonly ETCD_KMS_KEY_ID={{.ETCDKMSKeyID}}
+{{if .CloudKMSIntegration}}
+readonly CLOUD_KMS_INTEGRATION=true
+{{end}}
 `
-	kubeAPIServerManifestFileName = "kube-apiserver.manifest"
-	kmsPluginManifestFileName     = "kms-plugin-container.manifest"
-	kubeAPIServerStartFuncName    = "start-kube-apiserver"
+	/*
+		   encryptionProviderConfigForKMS is base64 encoded yaml below
```

Comment:
Could you base64-encode it at runtime? Makes it easier to edit.
The YAML itself can still be a string